qa/workunits/fs/snaps/snaptest-git-ceph.sh

```diff
@@ -15,10 +15,13 @@ timeout 1800 git clone git://git.ceph.com/ceph.git
 trap - EXIT
 cd ceph
 
-versions=`seq 1 21`
+versions=`seq 1 90`
 
 for v in $versions
 do
+    if [ $v -eq 48 ]; then
```

Comment:
What's special about number 48? If this is to skip a particular commit reset then this could be a randomized based skip?

Sorry, it isn't obvious to me :/

Reply:
Sorry forgot to mention it. There has no tag `48`, the test will fail when the `v == 48`, so I just skipped it.

Reply:
So, increasing the number of iterations *might* increase the likelihood of hitting bugs related to async creates,etc..? Do you see this in your (local) tests?

Reply:
Yeah, right.

Without my fixes about the async unlink/create I could see the kernel mounter would be stuck very easy. But I just once I see some files were deleted, which was the same with the failure we hit in qa test.